h/static/scripts/group-forms/components/CreateGroupForm.tsx

```diff
@@ -3,27 +3,52 @@ import { Button, Input, Textarea } from '@hypothesis/frontend-shared';
 export default function CreateGroupForm() {
   return (
     <>
-      <h1>Create a new private group</h1>
+      <h1 class="mt-[55px] mb-[30px] text-[#3f3f3f] text-[19px] leading-[15px]">
+        Create a new private group
+      </h1>
 
       <form>
-        <div>
-          <label for="name">Name*</label>
+        <div class="mb-[15px]">
+          <label for="name" class="text-[#7a7a7a] text-[13px] leading-[15px]">
+            Name<span class="text-[#d00032]">*</span>
+          </label>
           <Input id="name" autofocus autocomplete="off" required />
-          0/25
+          <div class="flex">
+            <div class="grow"></div>
```

Comment:
This flexbox stuff for right-aligning something is repeated a few times as well. Not sure if the duplication is worthy of an attempt to factor it out though.